sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerAccessor.java

```diff
@@ -113,6 +113,9 @@ private static SpannerAccessor createAndConnect(SpannerConfig spannerConfig) {
     SpannerOptions.Builder builder = SpannerOptions.newBuilder();
 
     Set<Code> retryableCodes = new HashSet<>();
+    // Add default retryable codes
+    retryableCodes.addAll(
+        builder.getSpannerStubSettingsBuilder().executeStreamingSqlSettings().getRetryableCodes());
```

Comment:
We are taking the retryable codes from the `executeStreamingSql` RPC here, and then apply that to the `executeStreamingSql` __and__ to all unary RPCs below. I don't think we should be doing the latter. Are there any specific unary RPCs that we want to set the retryable codes for? Or do we really want to set them for __all__ of them? E.g. this takes the default retryable codes of `ExecuteStreamingSql` and then applies them to RPCs like `Commit`.